### PR TITLE
New Version: RARLab.WinRAR version 7.00.0

### LINKED ISSUE #141711: [Update Request]: WINRAR version 7
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

RARLab.WinRAR

### Package Version

7.00

### Please describe the changes you would like to see

Winrar 7.00 has  been release: https://www.win-rar.com/start.html?&L=0

### PR DESCRIPTION
**Pull request by fancybody**
Resolves #141711
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/141724)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @fancybody,

> This PR's version number 7.0 has 2 parameters (sets of numbers between dots - major, minor, etc), which is less than the current manifest's version 6.24.0, which has 3 parameters.

Does this PR's version number **exactly** match the version reported in the Apps & features Settings page? (Feel free to attach a screenshot.)

(Automated response - build 857)

**Comment by fancybody** [comment]
@stephengillie
[Official Release News](https://www.win-rar.com/singlenewsview.html?&L=0) and [Download Links](https://www.rarlab.com/download.htm)

**Comment by stephengillie** [comment]
Verify:
WinRar 7.00 (64-bit) Apps and features entry.

File: manifests/r/RARLab/WinRAR/7.00.0/RARLab.WinRAR.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x64-700ro.exe
+  InstallerSha256: adc067acb1da85a5404e0c15e0bf7192bcb21c3b4c8ef789ba8f7b09c725c54b
+- InstallerLocale: ro-RO
+  Architecture: x86
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x32-700ro.exe
+  InstallerSha256: faf413a423bbd6539b220a4a207491fe28adcd0bd4d06e12f61808704931c589
+- InstallerLocale: ru-RU
+  Architecture: x64
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x64-700ru.exe
+  InstallerSha256: 3d7b6675e24cb33b24356bc0b8a9b366cff816f5bc392dd139684fe76fd8211b
```
Comment (stephengillie) [inline manifests/r/RARLab/WinRAR/7.00.0/RARLab.WinRAR.installer.yaml:161]:
```suggestion
  InstallerSha256: 6B62D2FC5A6119AA6292FF0E61E9BD87C26F23339E1B432A2FB73F898EC397D5
```

File: manifests/r/RARLab/WinRAR/7.00.0/RARLab.WinRAR.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+- InstallerLocale: ru-RU
+  Architecture: x86
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x32-700ru.exe
+  InstallerSha256: d7ec47d9b116d81bd994c2235d6e5b37b43e2f9cc864283e609daa04972caecf
```
Comment (stephengillie) [inline manifests/r/RARLab/WinRAR/7.00.0/RARLab.WinRAR.installer.yaml:165]:
```suggestion
  InstallerSha256: F22D3DAF7815C548F2DE0086DC4E4D2096C4A8AB2925E106BC9285197C05AC8A
```

File: manifests/r/RARLab/WinRAR/7.00.0/RARLab.WinRAR.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x32-700id.exe
+  InstallerSha256: 3cbeca9801dd8dbb4c96dab5a92e2332ee5e055a19603d1471938842b89f8710
+- InstallerLocale: mn-MN
+  Architecture: x64
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x64-700mn.exe
+  InstallerSha256: e0e3c5086431d441b3f93874ad513357526ea256f7989ebe6eb14d4d93ea5093
+- InstallerLocale: mn-MN
+  Architecture: x86
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x32-700mn.exe
+  InstallerSha256: 43837afce94b2f15a0708caa41920f861ed4d92fc4f9039dbfdc79c3ac16c36f
```
Comment (stephengillie) [inline manifests/r/RARLab/WinRAR/7.00.0/RARLab.WinRAR.installer.yaml:125]:
```suggestion
  InstallerSha256: DCDBD88EF7FF4BD94A076D136B76778F5C7D58F881203B2BC54460F8023DD5CB

```

File: manifests/r/RARLab/WinRAR/7.00.0/RARLab.WinRAR.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x64-700id.exe
+  InstallerSha256: c8018073b44b0fabf6b2a2ab050a6e08911c7a3370c427d97407367923cf14a5
+- InstallerLocale: id-ID
+  Architecture: x86
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x32-700id.exe
+  InstallerSha256: 3cbeca9801dd8dbb4c96dab5a92e2332ee5e055a19603d1471938842b89f8710
+- InstallerLocale: mn-MN
+  Architecture: x64
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x64-700mn.exe
+  InstallerSha256: e0e3c5086431d441b3f93874ad513357526ea256f7989ebe6eb14d4d93ea5093
```
Comment (stephengillie) [inline manifests/r/RARLab/WinRAR/7.00.0/RARLab.WinRAR.installer.yaml:121]:
```suggestion
  InstallerSha256: 8719CFD58C040FC0F3983B0B50D46B3E28B4350F551FD5A69415C63CC961D380
```

**Review by stephengillie — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**